pkg/runtime/runtime.go

```diff
@@ -593,7 +593,8 @@ func (a *DaprRuntime) sendToOutputBinding(name string, req *bindings.InvokeReque
 	if binding, ok := a.outputBindings[name]; ok {
 		ops := binding.Operations()
 		for _, o := range ops {
-			if o == req.Operation {
+			// ::TODO OperationKind provide a Equal method
+			if strings.EqualFold(string(o), string(req.Operation)) {
```

Comment:
@1046102779 as @pkedy mentioned in https://github.com/dapr/components-contrib/pull/880, most of the components have case sensitive operation names. Does this change need to be in the runtime?
I can see your example is for HTTP methods, can only the HTTP binding be changed and support both uppercase and lowercase(backwards compatibility) ? 

I am trying to understand why the comparison is changing in the runtime.

cc @pkedy

Reply:
I think it should be for HTTP component only.